app/src/components/modal/modal_deposit_withdraw/index.tsx

```diff
@@ -0,0 +1,226 @@
+      setIsTransactionModalOpen(false)
+    }
+  }
+
+  return (
+    <>
+      <Modal
+        isOpen={isOpen && !isTransactionModalOpen}
+        onRequestClose={onClose}
+        shouldCloseOnOverlayClick={true}
+        style={theme.fluidHeightModal}
+      >
+        <ContentWrapper>
+          <ModalNavigation>
+            <ModalNavigationLeft>
+              <IconArrowBack
+                hoverEffect={true}
+                onClick={() => {
+                  onBack()
+                  setDisplayFundAmount(new BigNumber(0))
+                  setAmountToDisplay('')
+                }}
+              />
+              <ModalTitle style={{ marginLeft: '16px' }}>{exchangeType} Dai</ModalTitle>
+            </ModalNavigationLeft>
+            <IconClose
+              hoverEffect={true}
+              onClick={() => {
+                onClose()
+                setDisplayFundAmount(new BigNumber(0))
+                setAmountToDisplay('')
+              }}
+            />
+          </ModalNavigation>
+          <ModalCard style={{ marginBottom: '16px', marginTop: '12px' }}>
+            <BalanceSection>
+              <BalanceItems>
+                <BalanceItem>
+                  <BalanceItemSide>
+                    <BalanceItemTitle>Wallet</BalanceItemTitle>
+                  </BalanceItemSide>
+                  <BalanceItemSide>
+                    <BalanceItemBalance style={{ marginRight: '12px' }}>{formattedDaiBalance} DAI</BalanceItemBalance>
+                    <DaiIcon size="24px" />
+                  </BalanceItemSide>
+                </BalanceItem>
+                <BalanceItem>
+                  <BalanceItemSide>
+                    <BalanceItemTitle>Omen Account</BalanceItemTitle>
+                  </BalanceItemSide>
+                  <BalanceItemSide>
+                    <BalanceItemBalance style={{ marginRight: '12px' }}>{formattedxDaiBalance} DAI</BalanceItemBalance>
+                    <DaiIcon size="24px" />
+                  </BalanceItemSide>
+                </BalanceItem>
+              </BalanceItems>
+            </BalanceSection>
+          </ModalCard>
+          <TextfieldCustomPlaceholder
+            formField={
+              <BigNumberInput
+                decimals={18}
+                name="amount"
+                onChange={(e: BigNumberInputReturn) => {
+                  setDisplayFundAmount(e.value)
+                  setAmountToDisplay('')
+                }}
+                value={displayFundAmount}
+                valueToDisplay={amountToDisplay}
+              ></BigNumberInput>
+            }
+            onClickMaxButton={() => {
+              const maxBalance = exchangeType === ExchangeType.deposit ? daiBalance : xDaiBalance || Zero
+              setDisplayFundAmount(maxBalance)
+              setAmountToDisplay(formatBigNumber(maxBalance, 18, 5))
+            }}
+            shouldDisplayMaxButton={true}
+            symbol={'DAI'}
+          ></TextfieldCustomPlaceholder>
```

Comment:
Small thing...i think that TextfieldcustomPlaceholder and bigNumber input can be self closing tag...